Cargo.toml

```diff
@@ -1,3 +1,3 @@
 [workspace]
 resolver = "2"
-members = ["packages/*", "tests/*"]
+members = ["packages/*"]
```

Comment:
Why "tests/*" is deleted?

Reply:
This tests folder is originally an integration test using contracts. Therefore, consider integration tests when adding examples/contracts.